_episodes/13-looking-up-data.md

```diff
@@ -75,7 +75,7 @@ The next exercise demonstrates this two stage process in full.
 {: .challenge}
 
 ## Reconciliation services
-Reconciliation services allow you to lookup terms from your data in OpenRefine against external services, and use values from the external services in your data. The official wiki provides [detailed information about this feature](https://github.com/OpenRefine/OpenRefine/wiki/Reconciliation).
+Reconciliation services allow you to lookup terms from your data in OpenRefine against external services, and use values from the external services in your data. The official User Manual provides [detailed information about this reconciling](https://docs.openrefine.org/manual/reconciling).
```

Comment:
Link should read "detailed information about this feature" (not "reconciling")

Reply:
@ostephens I reverted to "feature".

Might be better (in accessibility terms, to make hyperlink text meaningful) as something like "detailed information about reconciliation"? (just sounds poor in the paragraph since "reconciliation" is in every sentence...)

Reply:
@evanwill agreed that this phrasing is better - I've changed the link text to "detailed information about the reconciliation feature". Thanks